content-src/experiments/send.yaml

```diff
@@ -19,7 +19,7 @@ discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/send'
 privacy_notice_url: 'https://github.com/mozilla/send/blob/master/docs/metrics.md'
 measurements:
   - >
-    When you use Send, Mozilla receives an encrypted copy of the file you upload, and basic information about the file, such as filename and file size. Mozilla does not have the ability to access the content of your encrypted file, and only keeps it for the time or number of downloads indicated.
+    When you use Send, Mozilla receives an encrypted copy of the file you upload, and basic information about the file, such as filename, file hash and file size. Mozilla does not have the ability to access the content of your encrypted file, and only keeps it for the time or number of downloads indicated.
```

Comment:
Also needs an `npm run l10n:extract` run and the changes to en-US FTL added to the commit. (That's the failing test in CircleCI)